tests/legacy/bootstrap.php

```diff
@@ -44,6 +44,15 @@ public function __construct() {
 		$this->plugin_dir   = dirname( dirname( $this->tests_dir ) );
 		$this->wp_tests_dir = getenv( 'WP_TESTS_DIR' ) ? getenv( 'WP_TESTS_DIR' ) : sys_get_temp_dir() . '/wordpress-tests-lib';
 
+		if ( ! defined( 'WP_PLUGIN_DIR' ) ) {
+			define( 'WP_PLUGIN_DIR', $this->plugin_dir );
+		}
+
+		if ( file_exists( WP_PLUGIN_DIR . '/sample-woo-plugin.php' ) ) {
+			unlink( WP_PLUGIN_DIR . '/sample-woo-plugin.php' );
+		}
+		copy( __DIR__ . '/data/sample-woo-plugin.php', WP_PLUGIN_DIR . '/sample-woo-plugin.php' );
```

Comment:
How would you feel about doing this in the test's `setUp` and `tearDown` instead? This isn't something every test needs. I recognize this is likely done in a tmp directory, but it doesn't HAVE to!

Reply:
Makes sense, I have moved this logic to test's body instead of setUp though because not all tests will use this boilerplate. Ref: https://github.com/woocommerce/woocommerce/pull/26358/commits/f4e169f21e2c4bee3c17c0160ac2b1b4cc21cff5